src/components/main-content.tsx

```diff
@@ -22,13 +27,16 @@ export class MainContentComponent extends BaseComponent<IProps, IState> {
   }
 
   private renderDialog() {
```

Comment:
Probably time to change the name of this function.  renderSpace?  renderInvestigationContent?

Reply:
`renderSpace` and `renderInvestigationContent` seem confusing when we *are* rendering the dialog. Maybe just `renderMainContent`?

Reply:
I like it!  renderMainContent it is!